builtin_object.go

```diff
@@ -556,6 +556,46 @@ func (r *Runtime) object_setPrototypeOf(call FunctionCall) Value {
 	return o
 }
 
+func (r *Runtime) object_fromEntries(call FunctionCall) Value {
+	o := call.Argument(0)
+	r.checkObjectCoercible(o)
+
+	result := r.newBaseObject(r.global.ObjectPrototype, classObject).val
+
+	adderFn := func(call FunctionCall) Value {
```

Comment:
Why not inline this? Would also save the `.ToObject()` call.

Reply:
Good idea, done.